frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditEmail/EditEmail.tsx

```diff
@@ -130,10 +131,18 @@ export const EditEmail = ({ field }: EditEmailProps): JSX.Element => {
 
   const { data: form } = useCreateTabForm()
   const isPdfResponseEnabled = useMemo(
-    () => form?.responseMode !== FormResponseMode.Multirespondent,
-    [form],
+    () =>
+      form?.responseMode === FormResponseMode.Email ||
+      (form?.responseMode === FormResponseMode.Encrypt && includeFormSummary),
+    [form, includeFormSummary],
   )
 
+  const pdfResponseToggleDescription = useMemo(() => {
```

Comment:
same for this. this function does not seem to be computationally heavy operation

Reply:
move to todo. hotfixes should be kept to a minimal